robottelo/cli/factory.py

```diff
@@ -1046,27 +1046,23 @@ def make_host_collection(options=None):
 
     Options::
 
-        --content-host-ids CONTENT_HOST_IDS  List of content host uuids to be
-                                             in the host collection
         --description DESCRIPTION
         --host-collection-ids HOST_COLLECTION_IDS  Array of content host ids to
-                                                   replace the content hosts in
-                                                   host collection
-                                                   Comma separated list of vals
-
-        --max-content-hosts MAX_CONTENT_HOSTS Maximum number of content hosts
+                                              replace the content hosts in
+                                              host collection
+                                              Comma separated list of values
+        --hosts HOST_NAMES                    Comma separated list of values
+        --max-hosts MAX_CONTENT_HOSTS Maximum number of content hosts
                                               in the host collection
         --name NAME                           Host Collection name
         --organization ORGANIZATION_NAME
         --organization-id ORGANIZATION_ID     organization identifier
         --organization-label ORGANIZATION_LABEL
-        --unlimited-content-hosts UNLIMITED_CONTENT_HOSTS Whether or not the
-                                                          host collection may
-                                                          have unlimited
-                                                          content hosts
-                                                          One of true/false,
-                                                          yes/no, 1/0.
-         -h, --help                                       print help
+        --unlimited-hosts UNLIMITED_CONTENT_HOSTS Whether or not the host
+                                              collection may have unlimited
```

Comment:
Can you align the text with the beginning of the description? I know it will be a short column size, but this is to maintain the consistency across formatting.